src/sqlittaaja/html_style.py

```diff
@@ -0,0 +1,45 @@
+report_style = """
```

Comment:
Maybe this could be put into another CSS file? It's a bit hard to edit this huge string here.

Reply:
We have to be careful here because to my understanding PyInstaller doesn't bundle other assets by default. Something like [this](https://pyinstaller.org/en/stable/usage.html#cmdoption-add-data) should suffice. Make sure to test thoroughly that the final executable works.

Reply:
Built exe file and its reading now style.css. Updated publish.yml pyinstaller command to include this file.